### DISCUSSION TITLE
A Origem do Algodão Doce

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#306] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/a-origem-do-algodao-doce/

**Comment by one-eyed-willy** [comment]
Esse universo paralelo é meio doente

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/a-origem-do-algodao-doce/#comment_1114001344>):

> Esse universo paralelo é meio doente

Primeiro foi ~~limonada~~ [Humanada](<https://www.vish.blog.br///imagens/tirinhas/humanada>), depois [pizza](<https://www.vish.blog.br///imagens/tirinhas/noite-de-pizza>) e agora algodão-doce
